cli/http_util.rs

```diff
@@ -300,7 +300,10 @@ mod tests {
     let client = create_http_client();
     let result = fetch_string_once(client, &url, None).await;
```

Comment:
Since `fetch_string_once` no longer fetches string but rather bytes could you change its name to `fetch_once`?